### ISSUE TITLE
Language change not working

### ISSUE THREAD

**Issue by HenrikJannsen** [bisq-network/bisq2#2093]
A user reported that with a German OS he cannot switch to English as after restart its German again. 
Anyone with a German OS can confirm that?

**Comment by boldsuck** [comment]
Yes, I can confirm.
German Desktop Environment & cannot switch to English. I tried restarting bisq2 and rebooting OS.

```
Operating System: Debian GNU/Linux 12
KDE Plasma Version: 5.27.5
KDE Frameworks Version: 5.103.0
Qt Version: 5.15.8
Graphics Platform: X11
```

**Comment by HenrikJannsen** [comment]
Do you use latest Bisq 2 version 2.0.3?

**Comment by boldsuck** [comment]
Yes, more precisely the Debian installer: Bisq-2.0.3.deb

**Comment by HenrikJannsen** [comment]
Should be fixed with v2.0.4. It is available as pre-release at https://github.com/bisq-network/bisq2/releases/tag/v2.0.4 (if you enable notifications for pre-release in the preferences you get the update notification).
Can you try it out?
If you still have the issue can you post screenshot and exact description what you do? Could never reproduce the issue...

**Comment by boldsuck** [comment]
When I start bisq2 in the console, `~$ '/opt/bisq2/bin/Bisq 2'` everything is OK. I switched from German to English in bisq2 several times, and bisq2 started in the desired language.

If I start bisq2 in the KDE menu, bisq2 always starts in German. As a workaround, I modified my .desktop file:
`Exec=LANG=en_US.UTF-8 '/opt/bisq2/bin/Bisq 2'`

So if the problem doesn't occur for others, then it's due to some old KDE settings. This laptop has had several upgrades since KDE 3.n. For now, I see this as a problem solved or low priority.
